packages/ckb-indexer/package.json

```diff
@@ -15,8 +15,8 @@
     "test": "__tests__"
   },
   "dependencies": {
-    "@ckb-lumos/base": "0.18.0",
-    "@ckb-lumos/rpc": "0.18.0",
+    "@ckb-lumos/base": "0.20.0-alpha.3",
```

Comment:
Will we keep the `@nervina-labs/ckb-indexer` module?

Reply:
It can be removed, but I think it's possible to make as few changes as possible in the PR so that the reviewer can understand exactly what the changes in the PR are without too much effort

Reply:
the nervina indexer was removed